google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/StatementTimeoutTest.java

```diff
@@ -273,8 +272,12 @@ public void testTimeoutExceptionReadOnlyAutocommit() {
                 .build())) {
       connection.setReadOnly(true);
       connection.setStatementTimeout(TIMEOUT_FOR_SLOW_STATEMENTS, TimeUnit.MILLISECONDS);
-      expected.expect(SpannerExceptionMatcher.matchCode(ErrorCode.DEADLINE_EXCEEDED));
-      connection.executeQuery(Statement.of(SLOW_SELECT));
+      try {
+        connection.executeQuery(Statement.of(SLOW_SELECT));
+        fail("");
```

Comment:
nit: Here and below, please add a text like "Expected exception"

Reply:
Done